sema/bench/__main__.py

```
@@ -0,0 +1,108 @@
+# -*- coding: utf-8 -*-
```

Ruff

UTF-8 encoding declaration is unnecessary

Remove unnecessary coding comment

(UP009)